modules/tinymce/src/core/main/ts/annotate/Identification.ts

```diff
@@ -42,19 +45,25 @@ const identify = (editor: Editor, annotationName: Optional<string>): Optional<{u
 
 const isAnnotation = (elem: any): boolean => SugarNode.isElement(elem) && Class.has(elem, Markings.annotation());
 
+const isBogusElement = (elem: SugarElement<Node>, root: SugarElement<Node>) =>
+  Attribute.has(elem, 'data-mce-bogus') || SelectorExists.ancestor(elem, '[data-mce-bogus="all"]', isRoot(root));
```

Comment:
Should this be using `closest` incase the element itself has `data-mce-bogus="all"`?
```suggestion
  Attribute.has(elem, 'data-mce-bogus') || SelectorExists.closest(elem, '[data-mce-bogus="all"]', isRoot(root));
```

Reply:
Doesn't having `Attribute.has(elem, 'data-mce-bogus')` before check that? I guess it can't hurt though

Reply:
No you're correct... I'd compartmentalised it incorrectly as the left side checked `data-mce-bogus="1"` only. Feel free to ignore then.